src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleAssignment.java

```diff
@@ -30,7 +30,7 @@ public boolean isCaseRoleAssignment() {
             && !this.getAttributes().getCaseId().isEmpty();
     }
 
-    public boolean isAnExpiredRoleAssignment() {
+    public boolean isNotExpiredRoleAssignment() {
         final Instant machineTimestamp = Instant.now();
         return machineTimestamp.isAfter(beginTime) && machineTimestamp.isBefore(endTime);
```

Comment:
is it not possible to use attribute matcher for this.. or use this method in attribute matcher as well

Reply:
We would have to enhance CaseIdMatcher to include RoleType.CASE.name(), but it can have some implication on other parts of the code.